README.md

```diff
@@ -85,28 +89,30 @@ biocondaRepository := "/home/user/bioconda-recipes" // Where bioconda repository
 biocondaRecipeDir := "recipes" // Where the recipes will be created.
 // Defaults to target.value/recipes
 biocondaCommitMessage := "Auto" // The default commit message.
-// Default is "automated update for recipes of (name in Bioconda).value. 
+// Default depends on value of biocondaNewTool. 
+biocondaNewTool := true // Whether the tool is new in bioconda.
```

Comment:
Should this not be a option with as default None?

Reply:
It is a task. By default the plugin sets this value automatically depending on the updated repository.